scripts/fg_launcher_mac.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+#flightgear appimage setup script for flightgear-control project in Mac.
```

Comment:
`fgfs` is a mac binary rather than a linux app image so we should change the wording around that in this script.

Reply:
Addressed